__tests__/app.test.js

```diff
@@ -230,6 +230,74 @@ describe("GET /api/articles/:article_id", () => {
     });
   });
   
+  describe("PATCH /api/articles/:article_id", () => {
+    test("200: Updates the votes and responds with the updated article", () => {
+      const update = { inc_votes: 5 };
+  
+      return request(app)
+        .patch("/api/articles/1")
+        .send(update)
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.article).toEqual(
+            expect.objectContaining({
+              article_id: 1,
+              votes: expect.any(Number), 
+            })
+          );
+          expect(body.article.votes).toBeGreaterThan(0); 
```

Comment:
We should know what the votes are! Check the test data and add 5 to it and expect that